recipes/r-kdetrees/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set version = '0.1.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-kdetrees
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:     {{ cran_mirror }}/src/contrib/Archive/kdetrees/kdetrees_{{ version }}.tar.gz
+  sha256: af65c382b3e52a2bfe7ed8b9a3ff05f2cdd0a1a122c972ed2f79d02dae178ede
+
+build:
+  skip: true  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-ape
+    - r-distory
+    - r-ggplot2
+  run:
+    - r-base
+    - r-ape
+    - r-distory
+    - r-ggplot2
+
+test:
+  commands:
+    - $R -e "library('kdetrees')"           # [not win]
+
+about:
+  home: http://github.com/grady/kdetrees
+  license: GPL-2.0-only
+  summary: A non-parametric method for identifying potential outlying observations in a collection
+    of phylogenetic trees based on the methods of Owen and Provan (2011). Such discordant
+    trees may indicate problems with sequence annotation or tree reconstruction, or
+    they may represent interesting biological phenomena, such as horizontal gene transfers.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - eead-csic-compbio  
```

Please also add the conda-forge/r team:
```suggestion
    - conda-forge/r
    - eead-csic-compbio  
```

---

Thanks @dbast, I guess I need user @conda-forge/r to comment on this PR for the tests to pass?